<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prueba de styleSheet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242, 225, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSaludar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 0, 79);
font: 16pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);
border_color: 5px white;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Saludar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Arial&quot;;
color: rgb(0, 0, 118);</string>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 149, 255);
border-color: rgb(0, 98, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEdad">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 149, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Arial&quot;;
color: rgb(0, 0, 118);</string>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
